src/test/java/com/restassured/exercise/Skills.java

```diff
@@ -0,0 +1,104 @@
+package com.restassured.exercise;
+
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
+import static org.hamcrest.Matchers.is;
+import static org.testng.Assert.assertEquals;
+
+import java.util.List;
+
+import io.cucumber.java.en.Given;
+import  io.restassured.*;
+
+import org.json.simple.JSONObject;
+import org.junit.runner.Request;
+import org.testng.annotations.Test;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import io.restassured.specification.RequestSpecification;
+
+
+public class Skills {
+
+	//private JsonPath jsonPathEvaluator;
+
+	@Test
+
+	public void listSkills() {
+
+		RestAssured.baseURI = "https://parseapi.back4app.com";
+		given()
+			.headers("X-Parse-Application-Id",System.getenv("X-Parse-Application-Id"),"X-Parse-REST-API-Key",System.getenv("X-Parse-REST-API-Key"))
+			.get("/classes/Skills")
+		.then()
+			.statusCode(200)
+			.body("size()", greaterThan(0))
+		.log().all();
+
+		/*JsonPath jsonPathEvaluator = response.jsonPath();
```

Comment:
delete if is not necessary

Reply:
Changes made locally and will be loaded in the next PR.